lara-typescript/src/section-authoring/api/util/move-utils.spec.ts

```diff
@@ -0,0 +1,79 @@
+import { IPage, ISection, ISectionItem } from "../api-types";
+import { findSectionAddress, findSection, findSectionByAddress } from "./finding-utils";
+import { moveSection, ISectionDestination, RelativeLocation } from "./move-utils";
+import { makePages } from "./spec-helper";
+
+describe("moveSection", () => {
+
+  describe("Failing scenarios" , () => {
+    it ("it can't move a section to after a bogus section name", () => {
+      const pages = makePages(3);
+      const destination: ISectionDestination = {
+        relativeLocation: RelativeLocation.After,
+        destSectionId: "bogus" // not specifying a page...
+      };
+      const setPage = (nextPage: IPage) => null;
+      const sectionId = "section0";
+      const subject = moveSection({ destination, pages, sectionId, setPage });
+      // Move failed:
+      expect(subject).toBeFalsy();
```

Comment:
I'm not sure what `subject` means in this context. Perhaps `response` or `result` might be clearer? Or avoid the whole issue of naming with:
```suggestion
      expect(moveSection({ destination, pages, sectionId, setPage })).toBeFalsy();
```
Note that the same naming occurs several times below as well.